lib/config/hse_gparams.c

```diff
@@ -448,21 +449,25 @@ hse_gparams_set(
     const char *const               param,
     const char *const               value)
 {
+    struct params p;
+
     if (!params || !param || !value)
         return merr(EINVAL);
 
-    const struct params p = { .p_params = { .as_hse_gp = params }, .p_type = PARAMS_HSE_GP };
+    p = (struct params){ .p_params = { .as_hse_gp = params }, .p_type = PARAMS_HSE_GP };
```

Comment:
Why not in the declaration?

Reply:
Didn't really want to use parameters prior validation, but I can.

Reply:
Right, but odds are low validation will fail, and it's not a hot path...

Reply:
Ok, I will fix it.